src/App.js

```diff
@@ -74,87 +62,16 @@ export default class App extends React.Component<{}, AppState> {
                 'left'
             ],
             settings: {
-                dashSupport: this.appContext.bluetoothApiIsAvailable,
-                editorMode: 'block',
-                language: 'en'
+                language: 'en',
+                dashSupport: this.appContext.bluetoothApiIsAvailable
```

Comment:
Let's remove the `dashSupport` setting and always show the "Connect to Dash" control.

https://issues.fluidproject.org/browse/C2LC-91 will make the control disabled when Bluetooth isn't available